src/APIClient/Common.cpp

```diff
@@ -0,0 +1,9 @@
+#include <restclient-cpp/restclient.h>
+
+#include <APIClient/Common.hpp>
```

Comment:
The guidelines says that you need to put this one first, then a new line, then the others.